app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -79,6 +83,22 @@
 
     /**
      * @param input The {@link EditText} to evaluate.
+     * @return The resultcode of validateIban.
+     */
+    Constants.InputFieldResultCodes validateIban(final EditText input) {
+        final String iban = input.getText().toString();
+        if (EMPTY_STRING.equals(iban) || IBANVerifier.isValidIBAN(iban)) {
+            input.setBackgroundColor(Color.TRANSPARENT);
+            return CORRECT_FIELD;
+        } else if (!IBANVerifier.isValidIBAN(iban)) {
+            input.setBackgroundColor(this.alertColor);
+            return INVALID_IBAN;
+        }
+        return CORRECT_FIELD;
```

Comment:
Hmm... This line can never be reached, but yeah... The Java compiler doesn't know that it seems